testing/file_test/test_file.cpp

```diff
@@ -6,7 +6,10 @@
 
 #include <fstream>
 
+#include "common/check.h"
+#include "common/error.h"
```

Comment:
```suggestion
#include "common/error.h"
#include "common/raw_string_ostream.h"
```

(See RawStringOstream comment below)

Reply:
Done.